bin/GameIndex.yaml

```diff
@@ -5064,6 +5088,8 @@ SCUS-97374:
   region: "NTSC-U"
   gameFixes:
     - VU0KickstartHack # Fixes Character SPS (for both).
+    - EETimingHack # Fixes DMA errors 
```

Comment:
Dito :)

Reply:
Have you played through the USversion to make sure it isn't required?